lumicks/pylake/population/tests/test_dwelltimes.py

```diff
@@ -31,6 +31,44 @@ def test_likelihood(exponential_data):
     np.testing.assert_allclose(fit.bic, 4429.232045925579, rtol=1e-5)
 
 
+@pytest.mark.parametrize(
+    "data, min_obs, max_obs, loglik",
+    [
+        (np.array([0.2, 0.3, 0.6, 1.2]), 0.1, 1.4, -0.604631649126058),
+        (np.array([0.2, 0.3, 0.6, 1.2]), np.array([0.1, 0.1, 0.3, 0.3]), 1.3, 0.25784386656710745),
+        (np.array([0.2, 0.3, 0.6, 1.2]), 0.1, np.array([0.5, 0.5, 1.3, 1.3]), 1.1932900895106002),
+        (
+            np.array([0.2, 0.3, 0.6, 1.2]),
+            np.array([0.1, 0.1, 0.3, 0.3]),
+            np.array([0.5, 0.5, 1.3, 1.3]),
+            1.7155483581674074,
+        ),
```

Comment:
I'd be fine wrapping this with a `#fmt: off`

Reply:
👍